src/html/simple.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
Is this file also served by `gulp serve`? If so, please add a link from the playground to avoid it being forgotten